float64 velocity
float64 theta
float64 beta
float64 psi
float64 psi_dot
float64 x
float64 y
float64 x_dot
float64 y_dot
float64 radius
float64 lat_acc
float64 lon_acc
float64 yaw_rate
# normaly closed, i.e. 1 - ok, 0 - fault
bool emergency
